pkg/monitor/monitor/monitor_manager.go

```diff
@@ -75,12 +75,24 @@ func (m *MonitorManager) SyncMonitor(monitor *v1alpha1.TidbMonitor) error {
 	}
 
 	var firstTc *v1alpha1.TidbCluster
+	assetStore := NewStore(m.deps.ConfigMapLister, m.deps.SecretLister)
```

Comment:
The `ConfigMapLister` only watches the configmaps managed by TiDB Operator, and it cannot get all the configmaps in the cluster.

Reply:
https://github.com/pingcap/tidb-operator/blob/master/pkg/controller/dependences.go#L294 Otherwise, it may take too many memories for a large Kubernetes clusters.

Reply:
It seems that the configmap lister is not used.

Reply:
Currently, only secrets are used, but there will definitely be a need for configmap in the future. I have removed configmapLister.

Reply:
Do you mean user must add tidb-operator label to secret ,right?

Reply:
Add related label to the configmaps.